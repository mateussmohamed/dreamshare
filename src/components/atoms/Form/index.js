import styled from 'styled-components'

const Form = styled.form`
  @media only screen and (min-width: 320px) {
    display: flex;
    flex-direction: column;
    justify-content: center;
  }

  @media only screen and (min-width: 768px) {
    display: flex;
    flex-direction: row;
  }
`

export default Form
