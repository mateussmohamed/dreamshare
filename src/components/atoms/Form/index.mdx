---
name: Form
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import Form from './index'

# Form
<PropsTable of={Form} />

<Playground>
  <Form>

  </Form>
</Playground>
