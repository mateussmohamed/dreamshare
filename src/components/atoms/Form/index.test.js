import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import Form from './index'

describe('[<Form />]', () => {
  const wrap = (props = {}) => shallow(<Form {...props} />).dive()

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a Title' })

    expect(wrapper).toMatchSnapshot()
  })
})
