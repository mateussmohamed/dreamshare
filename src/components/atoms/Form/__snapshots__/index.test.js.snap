// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<Form />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "Form-nchkfv-0",
        "isStatic": true,
        "rules": Array [
          "@media only screen and (min-width:320px){display:flex;flex-direction:column;justify-content:center;}@media only screen and (min-width:768px){display:flex;flex-direction:row;}",
        ],
      },
      "displayName": "Form",
      "render": [Function],
      "styledComponentId": "Form-nchkfv-0",
      "target": "form",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a Title
</StyledComponent>
`;
